sdk/ts/orm/prisma.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: Prisma
+sidebarTitle: Prisma
+description: Configure Prisma to work with your Turso database
+---
+
+![Prisma banner](/images/guides/prisma-banner.png)
+
+## Prerequisites
+
+Before you start, make sure you:
+
+- [Install the Turso CLI](/cli/installation)
+- [Sign up or login to Turso](/cli/authentication#signup)
+- Prisma versions 5.4.2 and later
+
+<Steps>
+
+<Step title="Install the libSQL SDK and its Prisma driver">
+
+<CodeGroup>
+
+```bash npm
+npm install @libsql/client @prisma/adapter-libsql
+```
+
+```bash pnpm
+pnpm add @libsql/client @prisma/adapter-libsql
+```
+
+```bash yarn
+yarn add @libsql/client @prisma/adapter-libsql
+```
+
+</CodeGroup>
+
+</Step>
+
+<Step title="Configure database credentials">
+
+Get the database URL:
+
+```bash
+turso db show --url <database-name>
+```
+
+Get the database authentication token:
+
+```bash
+turso db tokens create <database-name>
+```
+
+Assign credentials to the environment variables inside `.env.local`:
+
+```bash
+TURSO_DB_URL="..."
+TURSO_DB_AUTH_TOKEN="..."
+```
+
+</Step>
+
+<Step title="Enable the `driverAdapters` Preview feature flag:">
+
+```js prisma/schema.prisma 3
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["driverAdapters"]
```

Comment:
Is this still a thing? I've seen several comments on Discord about `driverAdapters` not being a `previewFeatures` option anymore, but maybe their clients are outdated. Can we verify? It might also be worth asking to update [this page](https://www.prisma.io/docs/orm/overview/databases/turso) on the Prisma website too if that's the case.

Reply:
Went along with their reference for this guide, we could ask them for an update.